### PR TITLE
Add toolbar headline

### PR DESCRIPTION
**Pull request by egichuhis**
Hello there, kindly review my PR and advice.

Changes made
Add .gitignore file
Add markup for the toolbar and headline section
Add styling for both sections 
Add images for the toolbar menu and bg image headline section
Add README and LICENCE

portfolio

Thank you and cheers!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        <link rel="stylesheet" href="style.css">
+        <title>Edwin | Portfolio</title>
+    </head>
+
+    <body>
+        <header>
+            <section class="toolbar">
+                <span class="logo-container">
+                    <h3 class="logo-txt">Edwin G.</h3>
+                </span>
```
Comment (leonardodiasb) [inline index.html]: - [x] The logo usually is a link to the home page and should be wrapped with an `a` tag. Kindly change that.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                <span class="menu-btn">
+                    <img src="./imgs/Union.svg" alt="menu icon">
+                </span>
```
Comment (leonardodiasb) [inline index.html]: - [x] This is a navigation menu and should be wrapped with the `nav` tag for better semantics. Kindly change that.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            </span>
+            <span class="supporting">
+                I’m a Software Developer
+            </span>
+            <span class="description">
+                I can help you build a product , feature or website Look through some of my work and experience! If you
+                like what you
+                see and have a project you need coded,
+                don’t hesitate to contact me.
+            </span>
```
Comment (leonardodiasb) [inline index.html]: - [x] This is the title, subtitle, and paragraph and they should be wrapped with tags that are semantically correct for them, like the `h1` tag for the main title, `h2` or `h3` tag for the subtitle, and a `p` tag for a paragraph. Kindly change that.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @egichuhis, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Correct use of GitHub flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (egichuhis) [reply index.html]: Done

File: index.html (hunk 1 above)
Comment (egichuhis) [reply index.html]: Done.

File: index.html (hunk 3 above)
Comment (egichuhis) [reply index.html]: Done.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                <ul>
+                    <li><a href=""><i class="fa fa-github"></i></a></li>
+                    <li><a href=""><i class="fa fa-linkedin"></i></a></li>
+                    <li><a href=""><i class="fa fa-angellist"></i></a></li>
+                    <li><a href=""><i class="fa fa-twitter"></i></a></li>
+                    <li><a href=""><i class="fa fa-medium"></i></a></li>
+                </ul>
+            </span>
+        </section>
+    </body>
```
Comment (roshan-bajgain) [inline index.html:48]: - [ ] You should stick to the design as much as possible. please fix the spacing and also I would suggest you please download the social media icons from Figma and use it
### Your design
Annotation 2023-01-12 191533

### Figma design

fimga

**Review by roshan-bajgain — APPROVED** [review]
Hi @egichuhis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- GitHub flow is followed. ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**